lib/frizz/site.rb

```diff
@@ -9,10 +9,14 @@ def initialize(host, options={})
         @distribution = Distribution.new(@options[:distribution])
       end
 
-      local_options = options.select { |k, v| k == :redirect_rules }
+      local_options = options.select do |k, v|
+        k == :redirect_rules || k == :prefer_gzip
+      end
       @local = Local.new(path_to_deploy, ignorance, local_options)
 
-      remote_options = options.select { |k, v| k == :region }
+      remote_options = options.select do |k, v|
+        k == :region || k == :prefer_gzip
+      end
```

Comment:
@patbenatar 
I added a private method within the `Site` class for extracting key+value from a hash. Is the reasoning behind this to make things more readable, or allow for re-use elsewhere in the other files?